app/src/androidTest/java/com/github/se/project/ui/lesson/EditRequestedLessonTest.kt

```diff
@@ -235,3 +234,12 @@ class EditRequestedLessonTest {
     composeTestRule.onNodeWithText("10:00").assertExists()
   }
 }
+
+fun assertEnabledToBoolean(node: SemanticsNodeInteraction): Boolean {
+  try {
+    node.assertIsEnabled()
+  } catch (e: AssertionError) {
+    return false
+  }
+  return true
```

Comment:
I have two comments /questions regarding this method. First, is it not better to test if the not is displayed whether if it is enable? I do not see reasons whit enable is better than display. Secondly, is it not possible to use the property isEnable instead of doing this try catch block?